envergo/pages/templatetags/pages.py

```diff
@@ -188,9 +196,10 @@ def collapsible_menu(
 
 
 @register.simple_tag()
-def nb_available_depts():
+def nb_available_depts(site: Literal["haie", "amenagement"] = "amenagement"):
     """Return nb of depts where EnvErgo is available."""
-
+    if site == "haie":
```

Comment:
Sauf certains cas spécifiques, je trouve plus propre d'avoir une seule instruction `return`, ça permet d'éviter quelques bugs pernicieux. Je suggère par exemple la syntaxe suivante :

```suggestion
    Config = {
        "haie": HaieDepartmentConfig,
        "amenagement": MoulinetteConfig
    }.get(site)
    return Config.objects.filter(is_activated=True).count()
```